packages/web/src/components/DiscordCheck/DiscordButton.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
I just noticed this, but this directory should have the same name as the file; can you rename the parent `DiscordÇheck` to match `DiscordButton`?